MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -535,6 +535,16 @@ public string GetInputPathArgument(EncodingJobInfo state)
         {
             var mediaPath = state.MediaPath ?? string.Empty;
 
+            if (state.MediaSource.VideoType == VideoType.Dvd)
+            {
+                return _mediaEncoder.GetInputArgument(_mediaEncoder.GetPrimaryPlaylistVobFiles(state.MediaPath, null).ToList(), state.MediaSource);
+            }
+
+            if (state.MediaSource.VideoType == VideoType.BluRay)
+            {
+                return _mediaEncoder.GetInputArgument(_mediaEncoder.GetPrimaryPlaylistM2tsFiles(state.MediaPath).ToList(), state.MediaSource);
+            }
+
```

Comment:
this could be simplified as

```c#
return state.MediaSource.VideoType switch
{
	VideoType.Dvd => _mediaEncoder.GetInputArgument(_mediaEncoder.GetPrimaryPlaylistVobFiles(state.MediaPath, null).ToList(), state.MediaSource),
	VideoType.BluRay => _mediaEncoder.GetInputArgument(_mediaEncoder.GetPrimaryPlaylistM2tsFiles(state.MediaPath).ToList(), state.MediaSource),
	_ => _mediaEncoder.GetInputArgument(mediaPath, state.MediaSource)
};
```